connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -648,7 +636,6 @@ private static InputStream getInputStream(final CloseableHttpResponse response)
             @Override
             public void close() throws IOException {
                 response.close();
-                super.close();
             }
```

Comment:
Calling `response.close()` causes no connections to be released back to the pool.

https://java.net/jira/browse/JERSEY-3260